web/src/js/__tests__/ducks/tutils.js

```diff
@@ -8,4 +14,58 @@ export function createStore(parts) {
     )
 }
 
-export { default as TFlow } from './_tflow'
+export { TFlow }
+
+export function TStore(){
+    let tflow = new TFlow()
+    return mockStore({
+        ui: {
```

Comment:
Ok, your argument on seeing the structure is quite convincing. Let's continue as you proposed, if we see that this causes consistency issues in the long run, we could still change it. :)